straxen/common.py

```diff
@@ -376,6 +376,9 @@ def convert_channel(_data, replace=('channel', 'max_pmt')):
             if _rep not in data_keys:
                 # Apparently this data doesn't have the entry we want to replace
                 continue
+            if _dat['channel'].ndim != 1:
```

Comment:
Interesting that this works, bit surprised, what is different for NV, do they have a different structure?

Reply:
The problem is not the regular nveto data. Your function works fine if the channel range is outside the TPC channel range. However, lone_raw_records_statistics had also a channel field, but given its structure similar to pulse _counts the channel field is given by a nested array. So `channel` looks like [[0,1,2,3,4...], [0,1,2,3,4....]] while your function expects a single dimensional array.

Reply:
I see, thanks!